stdlib/public/Platform/glibc.modulemap.in.gyb

```diff
@@ -362,10 +363,12 @@ module SwiftGlibc [system] {
       header "@GLIBC_INCLUDE_PATH@/utmpx.h"
       export *
     }
+% if CMAKE_SDK != "FREEBSD":
```

Comment:
Negative tests are fragile and frustrating to read when porting to a new platform.  Could you change these if statements to use comparisons like `CMAKE_SDK in ['LINUX', 'ANDROID']`?